Gopkg.toml

```diff
@@ -67,3 +67,7 @@ required = [
     unused-packages = false
     non-go = false
     go-tests = false
+
+[[constraint]]
```

Comment:
Did you add this manually or dep upated Gopkg.toml?

Reply:
dep updated Gopkg.toml...